components/furnituremix/application/src/app/core/components/crystallize-components/dimensions-table.tsx

```diff
@@ -16,19 +16,25 @@ export const DimensionsTable = ({ dimensions }: { dimensions: any }) => {
                     className={`w-[20px] h-[20px] mr-4 ${open ? '-scale-y-100' : 'scale-y-100	'}`}
                 />
             </button>
-            <div className={`rounded-md ${open ? 'h-auto -mt-4 mb-10' : 'h-0 overflow-hidden'}`}>
-                <div>
-                    {dimensions?.map((dimension: any) => (
-                        <div key={dimension?.id} className="flex justify-between py-4 px-2 odd:bg-[#efefef]">
-                            <p className="font-semibold text-md">{dimension?.name}</p>
-                            <p className="text-md">
-                                {dimension?.content?.number}
-                                {dimension?.content?.unit}
-                            </p>
-                        </div>
-                    ))}
+
+            {open && dimensions && dimensions.length > 0 && (
```

Comment:
These blocks will be taken out of the DOM when its not clicked, hence it will not be indexed. Open should just control the height of the container like in the previous commit.

Reply:
We should use <details> and <summary> instead here